backtrace-library/src/main/java/backtraceio/library/metrics/SummedEvent.java

```diff
@@ -0,0 +1,37 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class SummedEvent extends Event {
+
+    @SerializedName("metric_group")
+    private final String name;
+
+    SummedEvent(String name) {
+        this(name, System.currentTimeMillis() / 1000, new HashMap<String, Object>());
+    }
+
+    SummedEvent(String name, long timestamp, Map<String, Object> attributes) {
+        super(timestamp);
+        this.name = name;
+        this.attributes = attributes;
+    }
+
+    SummedEvent(SummedEvent summedEvent) {
+        this(summedEvent.name, summedEvent.timestamp, summedEvent.attributes);
+    }
+
+    @Override
+    public String getName() {
+        return this.name;
+    }
+
+    public void addAttributes(Map<String, Object> attributes) {
+        if (attributes != null) {
```

Comment:
what if this.attributes will be null? probably exception will be thrown, is it?

Reply:
Ah good catch! Will improve this error handling!